src/show/views.py

```diff
@@ -76,3 +78,32 @@ def post(self, request, pk):
             show.save()
 
         return success_response(self.serializer_class(show, context={"request": request}).data)
+
+
+class AddToListView(generics.GenericAPIView):
+
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def post(self, request, pk):
+        data = json.loads(request.body)
+        list_ids = data.get("list_ids")
+        if not Show.objects.filter(pk=pk):
+            return failure_response(f"show with id {pk} does not exist")
+
+        show = Show.objects.get(pk=pk)
+        success_lists = []
+        for lst_id in list_ids:
```

Comment:
you can get all possible lists with `Lst.objects.filter(pk__in=list_ids)` which is a lot faster